### ISSUE TITLE
Ability to specify font for menu "keys"

### ISSUE THREAD

**Issue by macintacos** [VSpaceCode/vscode-which-key#80]
## Problem

For example, if the `l` and `I` menu options are next to each other, because the default sans-serif font displays the keys, they look remarkably identical to one another.

## Solution

I know I can (probably?) solve it with CSS, but it'd be great if it were an option to set your own font for the menu items. Personally, I'd set it to a monospace font, since this is basically a solved problem with those for the most part.

## Alternatives

Can use CSS - I will in the meantime, but it'd be nice if it were supported in the extension itself

**Comment by MarcoIeni** [comment]
Switching `l` with `I` is the story of my life (take a look at my surname 😅)
Imo we could add the font as a setting similar to how vscode does. Not sure how feasible it is.
![image](https://github.com/VSpaceCode/vscode-which-key/assets/11428655/00bf9b8a-8900-4ea4-b9c8-5e8c796e5938)

**Comment by macintacos** [comment]
Yeah I took a stab at trying to use CSS to set it, and unfortunately it's not quite so straightforward since the component used seems to be a generic one. Every class I've tried to set seems to impact something else somewhere else in the UI that I don't want touched - literally just want the labels to be monospace and nothing else (even if other "menus" are invoked that use the same generic component).

I'm not sure what you can do here since this might be a VSCode limitation - is it possible to add CSS classes for the rows in the quick input panel that users can more readily target?

**Comment by stevenguh** [comment]
<img width="530" alt="image" src="https://github.com/VSpaceCode/vscode-which-key/assets/3455662/09bbbc96-f148-4387-aeb3-2e02657b7a23">

We have a setting which uses [fullwidth characters in CJK](https://en.wikipedia.org/wiki/Halfwidth_and_Fullwidth_Forms_(Unicode_block)) to workaround the spacing issue. We can consider adding other options such as [Mathematical Alphanumeric Symbols](https://www.unicode.org/charts/PDF/U1D400.pdf) which often has a built-in "style" to the character.

**Comment by macintacos** [comment]
Personally, I'd like to be able to specify a font of my choice, but if that's not possible then I'd gladly accept whatever would help disambiguate all characters used for keys.

**Comment by stevenguh** [comment]
Since we are using VScode menu API, I doubt they would allow special customization any time soon.

But there is some quick result using math [monospace character block](https://en.wikipedia.org/wiki/Mathematical_Alphanumeric_Symbols)

<img width="608" alt="image" src="https://github.com/VSpaceCode/vscode-which-key/assets/3455662/b5daba7b-0aa3-4c21-9484-48e3aef68c37">

**Comment by fgimian** [comment]
As an additional question to this, is there a way to set the font and font size for the menu? I do notice the text is quite a lot smaller and looks different to the usual Ctrl+Shift+P and Ctrl+P menus. 😄 

Cheers
Fotis
